examples/shopping_ads/add_shopping_smart_ad.rb

```diff
@@ -240,9 +240,8 @@ def add_default_shopping_listing_group(client, customer_id, ad_group_name)
       options[:merchant_center_id] = v
     end
 
-    opts.on('-d', '--create-default-listing-group CREATE-DEFAULT-LISTING-GROUP',
-        String, 'Create Default Listing Group') do |v|
-      options[:create_default_listing_group] = true if v
+    opts.on('-d', '--[no-]create-default-listing-group', 'Create Default Listing Group') do |v|
```

Comment:
We have to conform to the exact format used in other languages, so that the functional testing framework doesn't have to change its implementation just for the Ruby case. Unfortunately, even though this is fairly elegant, we have to stick to a single `--create-default-listing-group` arg which then accepts a true/false value.

Reply:
Updated